pkg/process/procutil/process_linux_test.go

```diff
@@ -165,6 +165,13 @@ func testProcessesByPID(t *testing.T) {
 			compareProcess(t, ConvertFromFilledProcess(expectProc), procByPID[pid])
 		}
 	}
+
+	// Test processesByPID with collectStats == false
+	procByPID, err = probe.ProcessesByPID(time.Now(), false)
+	assert.NoError(t, err)
+	for _, proc := range procByPID {
+		assert.Empty(t, proc.Stats)
```

Comment:
We're currently testing that only `stats` are empty. Shouldn't we also assert that stats are empty _and_ metadata fields were correctly populated?

You could probably pass a new parameter to the `compareProcess` function to decide whether or not we're comparing the `stats` between `procV1` and `procV2` and replicate the logic above to also exclude processes without command line.

Reply:
Yeah that's a smart idea. We need to make sure that the tests come out properly